clients/guzzle5-adapter.rst

```diff
@@ -1,2 +1,52 @@
 Guzzle5 Adapter
 ===============
+
+An HTTPlug adapter for the `Guzzle 5 HTTP client`_.
+
+Installation
+------------
+
+To install the Guzzle adapter, which will also install Guzzle itself (if it was
+not yet included in your project), run:
+
+.. code-block:: bash
+
+    $ composer require php-http/guzzle5-adapter
+
+.. include:: includes/install-message-factory.inc
+
+.. include:: includes/install-discovery.inc
+
+Usage
+-----
+
+Begin by creating a Guzzle client, passing any configuration parameters you
+like::
+
+    use GuzzleHttp\Client as GuzzleClient;
+
+    $config = [
+        // Config params
+    ];
+    $guzzle = new GuzzleClientClient($config);
+
+Then create the adapter::
+
+    use Http\Adapter\Guzzle5\Client as GuzzleAdapter;
+    use Http\Message\MessageFactory\GuzzleMessageFactory;
+
+    $adapter = new GuzzleAdapter($guzzle, new GuzzleMessageFactory());
+
+Or if you installed the :ref:`discovery <discovery>` layer::
```

Comment:
did you define a label discovery somewhere? if you want to link to the document discovery.rst, its `:doc:`. `:ref:` is for linking to labels within some page (anchor links in the same or another page).